Puppetfile

```diff
@@ -22,6 +22,7 @@ mod 'theforeman/puppet',             :git => 'https://github.com/theforeman/pupp
 mod 'katello/foreman_proxy_content', :git => 'https://github.com/theforeman/puppet-foreman_proxy_content'
 mod 'katello/certs',                 :git => 'https://github.com/theforeman/puppet-certs'
 mod 'katello/katello',               :git => 'https://github.com/theforeman/puppet-katello'
+mod 'puppet/redis',                  :git => 'https://github.com/voxpupuli/puppet-redis', :ref => 'd5008dd41e92a66323c854b220c197425fdc7232'
```

Comment:
For now until we get a release of the module

Reply:
```suggestion
mod 'puppet/redis',                  '>= 8.5.0'
```